src/scripts/templates.test.js

```diff
@@ -40,11 +56,7 @@ describe('templates', () => {
             searchEnabled,
             passedElementType,
           );
-
-          expect(getType(actualOutput)).to.equal('HTMLDivElement');
-          expect(stripElement(actualOutput)).to.equal(
-            stripElement(expectedOutput),
-          );
+          expectEqualElements(actualOutput, expectedOutput);
```

Comment:
```suggestion
          expectEqualElements(expectedOutput, actualOutput);
```

Reply:
If you have them this way round, all the attributes of the expected output will be checked as you're looping over `element1.attributes` above

Reply:
I’m comparing first that two elements have the same number of attributes - after that it’s not important which one to loop